atomic_defi_design/Dex/Settings/SettingModal.qml

```diff
@@ -527,8 +527,8 @@ Qaterial.Dialog
                                 {
                                     Layout.alignment: Qt.AlignVCenter
                                     text: API.app.settings_pg.get_version()
-                                    onCopyNotificationTitle: qsTr("DEX Version")
-                                    onCopyNotificationMsg: qsTr("DEX Version copied to clipboard.")
+                                    onCopyNotificationTitle: qsTr("AtomicDEX Desktop Version")

```

Comment:
Be careful, use API variable to get application name since we have branded dex

Reply:
Good catch

Reply:
I've reverted to just using "Application version" because using app variable led to header text overflowing outside the notification